bin/start.sh

```diff
@@ -58,10 +62,23 @@ if [ ! -f $INSTALLATIONCONFIG ] && [[ $SKIP_INSTALL_CHECK -eq 0 ]]; then
 OPTIONAL
 fi
 
+# If DO_NOT_DAEMONIZE is declared, use the log4j config that outputs
+# to stdout/stderr
+if [[ $DO_NOT_DAEMONIZE -eq 1 ]]; then
+    LOGCONFIG="conf/logs/log4j2.properties"
+fi
+
 echo "starting loklak"
 echo "startup" > $STARTUPFILE
 
-cmdline="$cmdline -server -classpath $CLASSPATH -Dlog4j.configurationFile=$LOGCONFIG org.loklak.LoklakServer >> data/loklak.log 2>&1 &";
+cmdline="$cmdline -server -classpath $CLASSPATH -Dlog4j.configurationFile=$LOGCONFIG org.loklak.LoklakServer";
+
+# If DO_NOT_DAEMONIZE, pass it to the java command, end of this script.
+if [[ $DO_NOT_DAEMONIZE -eq 1 ]]; then
+    exec $cmdline
+fi
+
+cmdline="$cmdline >> data/loklak.log 2>&1 &"
 
 eval $cmdline
```

Comment:
@niccokunzmann No, it adds `>> data/loklak.log 2>&1 &" which redirects `stdout` and `stderr` in case if loklak crashed before log4j2 and the `&` symbol at the end is what actually makes it run daemonized